assets/stylesheets/components/_notification-banner.scss

```diff
@@ -15,6 +15,32 @@
   // scss-lint:enable SelectorFormat
 }
 
+.notification-banner__hiviz {
+  background-color: tint($yellow, 25%);
+
+  a {
```

Comment:
Also for discussion. An element descendent selector!

Reason: any links within this class need tweaking for AA contrast.

I'm loath to have to pepper `.util-hi-contrast__yellow` or similar all over the place. This feels a bit cleaner?